src/System.Management.Automation/engine/MergedCommandParameterMetadata.cs

```diff
@@ -228,7 +228,7 @@ private int AddParameterSetToMap(string parameterSetName)
                 // A parameter set name should only be added once
                 if (index == -1)
                 {
-                    if (_nextAvailableParameterSetIndex == uint.MaxValue)
+                    if (_nextAvailableParameterSetIndex == 32)
```

Comment:
Please define a constant for 32 with comment why we use the value.

Reply:
Good call. Better in the latest commit? I also switched to `>=` mostly for readability though it **should** never get above 32.